src/nurse_ussd.js

```diff
@@ -175,558 +152,359 @@ go.app = function() {
         });
 
 
-        self.add('states_start', function(name) {
+    // DELEGATOR START STATE
+
+        self.add('isl_route', function(name) {
+            // reset working_on extra
+            self.user.extra.working_on = "";
+            return self.im.contacts
+                .save(self.user)
+                .then(function() {
+                    if (self.contact.extra.is_registered === 'true') {
```

Comment:
I think we need to namespace this for a different NurseConnect is_registered flag. I've seen mistakes that nurses have made where they self register on the lines.